instagram/src/App.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import { PostContainer } from './components/PostContainer';
+import postData from './application-data';
+import LikeButton from './components/LikeButton';
+import NavBar from './components/NavBar';
+import './App.css';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      postData: [],
+      comments: []
+    };
+  }
+
+  componentDidMount() {
```

Comment:
Doing this is fine, it won't break anything, but it also isn't actually necessary. You can just hand off the postData into the `PostContainer` component directly without having it be placed in this component's state.